File: assistant/ai-assistant-api-old-python/models/community.py
from typing import Union, List, Optional

from pydantic import BaseModel

class Community(BaseModel):
    community_id: Optional[int] = None
    name: str
    description: str
    language: str
    image_url: Optional[str] = None
    counter_posts: Optional[int] = 0
    counter_points_for: Optional[int] = 0
    counter_points_against: Optional[int] = 0
    domain_id: int
    cluster_id: Optional[int] = None
    created_at: str
    updated_at: str
    status: str
    assistantConfiguration: str

File: server_api/ts-out/server_api/active-citizen/engine/assistant/ai-assistant-api-unused/models/community.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const typescript_optional_1 = require("typescript-optional");
const pydantic_1 = require("pydantic");
class Community extends pydantic_1.BaseModel {
    community_id = typescript_optional_1.Optional.empty();
    name;
    description;
    language;
    image_url = typescript_optional_1.Optional.empty();
    counter_posts = typescript_optional_1.Optional.of(0);
    counter_points_for = typescript_optional_1.Optional.of(0);
    counter_points_against = typescript_optional_1.Optional.of(0);
    domain_id;
    cluster_id = typescript_optional_1.Optional.empty();
    created_at;
    updated_at;
    status;
    assistantConfiguration;
}
